### DISCUSSION TITLE
Chat is down?

### DISCUSSION THREAD

**Discussion by hubchart** [community/community#106907] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Is copilot chat down? its not working

**Comment by Akash1134** [comment]
Hi @hubchart ,

There are no ongoing incidents for Copilot. Could you please elaborate your issue in detail?

To get started, please check out our Copilot Chat [troubleshooting guide](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-issues-with-github-copilot-chat).

**Comment by davecheney** [comment, marked as answer]
@hubchart please review this update https://github.com/orgs/community/discussions/101438 and ensure you're using the latest version of the chat extension. You may need to also update your editor.

**Comment by ramesh2051** [comment]
Hi @hubchart To ensure Copilot Chat continues working from February 1, 2024, please update to the most recent version of the Copilot extension and ensure your firewall and network settings allow communication to https://api.githubcopilot.com.
